homeassistant/components/wemo/manifest.json

```diff
@@ -14,5 +14,8 @@
   },
   "codeowners": ["@esev"],
   "iot_class": "local_push",
-  "loggers": ["pywemo"]
+  "loggers": ["pywemo"],
+  "supported_brands": {
+    "digital_loggers": "Digital Loggers"
```

Comment:
This PR is missing a reference PR against or brands repository adding this brand.
Could you open one?

Reply:
Thanks!